app/studentsAPI.js

```diff
@@ -0,0 +1,16 @@
+const studentsDB = require('./studentsDB');
+
+var studentsAPI = {
+    find: function (id, response) {
+        const data = studentsDB.find(student => student.id === id);
+        
+        if(data) {
+            response.status(200).send(data);
+        }
+        else {
+            response.sendStatus(400);
```

Comment:
Leia a seção "[Use HTTP Response Codes to Indicate Status](http://www.restapitutorial.com/lessons/restquicktips.html)"